.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+# Upload a new project
```

Comment:
Change "Upload" to "Integrate"

Reply:
Yes, you are right. I am still adding more functions to it.

Reply:
Cool!
GitHub supports multiple PR templates, which could be suitable for our case.

Reply:
Removed PR templates, added criticality score in the comments